index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alan x Walkers - Unity- YouTube</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link rel="stylesheet" href="index.css" />
+        
+</head>
+<body>
+    <nav>
+        <div>
+            <i class="material-icons dp48">dehaze</i>
+            <img class="brand" src="./images/YouTube-Logo.wine.svg"  alt="YouTube Logo"/>
+        </div>
```

Comment:
- [ ] The YouTube logo is not showing up on your clone. It appears as though there is no images folder in your clone so the images in that folder will not appear on your page. Please ensure to commit the images folder. 

| Source Page	| Clone Page |
|:---:|:---:|
|![Screen Shot 2020-07-13 at 7 49 00 PM](https://user-images.githubusercontent.com/48804258/87336300-e8f2e500-c541-11ea-8986-cf21958d946b.png)|![Screen Shot 2020-07-13 at 7 48 31 PM](https://user-images.githubusercontent.com/48804258/87336256-d5e01500-c541-11ea-9c28-cd8b6b91007e.png) |